vi/lessons/basics/pipe-operator.md

```diff
@@ -1,70 +1,69 @@
 ---
 layout: page
-title: Pipe Operator 
+title: Pipe Operator
```

Comment:
@kiennt there's a few titles that weren't translated like this, was that intentional?